Feature/make susept base rates female not avg

e-lipman opened the pull request:
I meant to make these changes on a different branch, but this is where the updates to the risk adjustment for comorbidities are

Commit:
feat: update many of the hospitalization and death ORs

Commit:
Add new excel workbook with calculations of hospitalization and death ORs for comorbidities, using updated data (June 2020)

Commit:
refactor: rename older OR calculation spreadsheet (April 2020)

Commit:
doc: Add excel file with some calculations on suspectibility by age and sex, not currently used in app since not adjuseted for undereporting

Commit:
refactor: create folder for validation

Commit:
feat: Change the way suseptibility is inflated based on comorbidities. Taking into account new mutually adjusted ORs and making exceptions for ORs that are not mutually adjusted

Commit:
fix: If statement for case where 0 conditions

Commit:
fix: inequality sign in incorrect direction

Commit:
doc: Update methods section with new links for risk factors

Commit:
Update suseptibility validation - compare numbers from new suseptibility calculation to COVER app

Commit:
Update suseptibility validation - compare numbers from new suseptibility calculation to COVER app

Commit:
fix merge

Merge branch 'feature/make_susept_base_rates_female_not_avg' of https://github.com/mathematica-mpr/covid_risk_score into feature/make_susept_base_rates_female_not_avg

# Conflicts:
#	Validation/suseptability_validation.xlsx

Commit:
fix: correct comment

xindyhu commented on src/global_var.R:
```diff
@@ -35,14 +35,23 @@ death_list = c(0.00161, 0.00695, 0.0309,  0.0844,  0.161,   0.595,   1.93,   4.2
+cvd_or      = c(1.4, 4.88, 1.27)
+diabetes_or = c(3.1, 4.57, 1.79)
+hyper_or    = c(1.1, 4.57, 1) # use the same as diabetes
+smoker_or   = c(2.67, 2.64, 1.12) # death OR not available in CCDC
```
Smoker hospitalization OR is 2.3 in the June2020_new_comorbidity_OR.xlsx

Commit:
fix: correct hosp OR for smoker, immuno, other - to match spreadsheet

Commit:
fix: Correct number for death "other" OR, which should be same as val from April

xindyhu commented on src/results.R:
```diff
@@ -54,12 +54,25 @@ calculateRisk <- function(input, county_data) {
+    }
   }
   if (input$gender == "male") {
     hosp_odds = hosp_odds * male_or[1] # should be hosp_odds(female) * male_or[1]
```
@e-lipman Do we need to change here to female hospitalization odds? Or is it already taken care of else where?

e-lipman replied:
@xindyhu good question. hosp_odds, icu_odds, and death_odds are initially set to the female odds in lines 49-51, and then the odds are multiplied up in he same manner as before. So I believe this is correct

xindyhu replied:
@e-lipman Got it, that's perfect, thanks!

Commit:
fix: Updated outdated comment that caused confusion in review of yesterday's pull request

Commit:
Pull from develop. Resolve merge conflist with new sources

Merge branch 'develop' of https://github.com/mathematica-mpr/covid_risk_score into feature/make_susept_base_rates_female_not_avg

# Conflicts:
#	src/global_var.R
